README.md

```diff
@@ -1,199 +1,199 @@
 
 ### 2. Application Review
 
-   1. Curators can (and usually do) issue comments and request changes on the pull request.
-   2. Clarifications and amendments made in the comments _need to be included in the application_. You may address feedback by directly modifying your application and leaving a comment once you're done. Generally, if you don't reply within 2 weeks, the application will be closed due to inactivity, but you're always free to reopen it as long as it hasn't been rejected.
-   3. When all requested changes are addressed, someone will mark your application as `ready for review` and share it internally with the rest of the committee.
-   4. The application will be accepted and merged as soon as it receives **4 approvals**, or closed after two weeks of inactivity. Unless specified otherwise, the day on which it is accepted will be considered the starting date of the project, and will be used to estimate delivery dates.
+1. Curators will review and comment on your pull request.
+2. Address feedback by modifying your application.
+3. Once ready, the application will be marked as `ready for review` and shared with the committee.
+4. Approval requires **4 votes** and will mark the project start date.
 
 ### 3. Milestone Delivery and Payment
 
-   Milestones are to be delivered on the [Grant Milestone Delivery](https://github.com/smart-contract-bounty/Grant-Milestone-Delivery) repository following the process described therein.
-
-### Changes to a Grant after Approval
-
-- Accepted grant applications can be amended at any time. However, this _necessitates a curators reevaluation_ and the same number of approvals as an application. If your application has been accepted and, during development, you find that your project significantly deviates from the original specification, please open a new pull request that modifies the existing application. This also applies in case of significant delays.
-- If your _delivery schedule_ significantly changes, please also open a pull request with an updated timeline.
-- If your deliveries are significantly delayed and we cannot get a hold of you, we will terminate the grant (4 approvals required. If Curator creates the termination PR, only 3 more approvals are required).
+Milestones are to be delivered in the [Grant Milestone Delivery](https://github.com/smart-contract-bounty/Grant-Milestone-Delivery) repository.
 
+---
 
-## :bulb: Help
-### Real-time conversation
+**💡 Help**
 
-We have Matrix/Element channels for real-time support regarding running bounties. Join the conversation.
+For real-time support, join:
 
 - [Ecosystem Grants Community](https://matrix.to/#/#sm-bounty-support:matrix.org)
 
-Other channels
+Additional channels:
+
 - [Smart contracts & Parity's ink!](https://matrix.to/#/#ink:matrix.parity.io)
-- [Brushfam ink! community](https://matrix.to/#/!utTuYglskDvqRRMQta:matrix.org?via=matrix.org&via=t2bot.io&via=web3.foundation)
+- [Brushfam ink! community](https://discord.gg/9FRETSPmp9)
```

Comment:
This looks abandoned.

Reply:
Indeed.
https://x.com/0xMarkian/status/1702384661605867759

Reply:
BTW, Pendzl is successor of the Openbrush library. It is audited and compatible with ink! 5.0.0 and was granted by W3F.

https://github.com/Pendzl/pendzl